test/utils/testHelper.ts

```diff
@@ -1,47 +1,50 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-const sinon = require("sinon");
-const { AppConfigurationClient } = require("@azure/app-configuration");
-const { ClientSecretCredential } = require("@azure/identity");
-const { SecretClient } = require("@azure/keyvault-secrets");
-const uuid = require("uuid");
+import * as sinon from "sinon";
+import { AppConfigurationClient } from "@azure/app-configuration";
+import { ClientSecretCredential } from "@azure/identity";
+import { KeyVaultSecret, SecretClient } from "@azure/keyvault-secrets";
+import * as uuid from "uuid";
 
 const TEST_CLIENT_ID = "62e76eb5-218e-4f90-8261-000000000000";
 const TEST_TENANT_ID = "72f988bf-86f1-41af-91ab-000000000000";
 const TEST_CLIENT_SECRET = "Q158Q~2JtUwVbuq0Mzm9ocH2umTB000000000000";

```

Comment:
Just to confirm - these aren't real IDs right?

Reply:
Fake IDs with same shape/length of real ones. Or maybe it’s better/clearer to replace all characters with zeros?

Reply:
Yes all zeros would be clearer.

Reply:
updated.